src/SWP/Bundle/FixturesBundle/DataFixtures/PHPCR/LoadArticlesMediaData.php

```diff
@@ -59,6 +59,14 @@ public function loadArticles($env, $manager)
                     'locale' => 'en',
                 ],
             ],
+            'dev' => [
+                [
+                    'title' => 'Test news article',
```

Comment:
We need more articles on dev/demo - and with more realistic titles (not like this). At least 3-4 articles per route.